src/ape/managers/project/types.py

```diff
@@ -194,8 +194,15 @@ def create_manifest(
                 name=self.name,
                 version=self.version,
             )
+            del manifest.contract_types
```

Comment:
Do we need this `del` statement? Does it hurt if there are contracts?
The point is that we don't want to compile at this stage, but they are already compiled, it should be harmless to include them.